lib/xcodeproj/project/object/root_object.rb

```diff
@@ -76,9 +80,19 @@ def ascii_plist_annotation
           ' Project object '
         end
 
+        def to_hash_as(method = :to_hash)
+          hash_as = super
+          if !hash_as['packageReferences'].nil? && hash_as['packageReferences'].empty?
```

Comment:
@segiddins these values are "optional" as in a default Xcode 11 project will *not* include them in its plist. Is there a better way in Xcodeproj to handle this? 

The way I have it here is passing all tests but felt its a bit ugly. Alternatively, we can update all tests to ensure `packageReferences => []` is taken into consideration even though they are none in the actual `.pbxproj.`